openshift/template.yaml

```diff
@@ -80,13 +80,15 @@ objects:
                 - name: AWS_SQS_ACCESS_KEY_ID
                   valueFrom:
                     secretKeyRef:
-                      name: aws
-                      key: sqs-access-key-id
+                      name: aws-sqs
+                      key: aws_access_key_id
                 - name: AWS_SQS_SECRET_ACCESS_KEY
                   valueFrom:
                     secretKeyRef:
-                      name: aws
-                      key: sqs-secret-access-key
+                      name: aws-sqs
+                      key: aws_secret_access_key
+                - name: REFRESH_INTERVAL
```

Comment:
Is this going to be a static value? If not, you use something like CPU_REQUEST that can be handled via saas-analytics PR directly

Reply:
We dont need this value now. Need to remove this later. the centos build is failing because its expecting this value. Will change it during saas analytics PR and then change here later

Reply:
https://github.com/openshiftio/saas-analytics/blob/master/bay-services/github-refresh-cronjob.yaml